tests/fakes.py

```diff
@@ -56,7 +56,7 @@ def get_branch_sha(self, org, repo, branch):
     def get_tag_sha(self, org, repo, tag):
         return "test_sha"
 
-    def get_file(self, org, repo, branch, filepath=None):
+    def get_file(self, org, repo, branch, filepath="project.yaml"):
```

Comment:
Yes.

There is a trade-off though between making the PR as easy to review as possible and the amount of effort spent making it so and how legible the commit log is in future (many micro-commits that only exist to marginally ease review can make it harder to read and understand that log).

Also it could be confusing to a reviewer and take some time to parse a commit that changes a test to expected to fail to set up subsequent commits.

I think this commit clearly explains itself in its commit message and I thought that was sufficient.